package.json

```diff
@@ -56,12 +56,12 @@
     "@fortawesome/free-regular-svg-icons": "6.6.0",
     "@fortawesome/free-solid-svg-icons": "6.6.0",
     "@fortawesome/react-fontawesome": "0.2.2",
-    "@openedx/paragon": "^22.0.0",
     "js-cookie": "^3.0.1",
     "lodash": "^4.17.21"
   },
   "peerDependencies": {
     "@edx/frontend-platform": "^7.0.0 || ^8.0.0",
+    "@openedx/paragon": ">= 21.11.3 < 23.0.0",
```

Comment:
[sanity check] is there anything in the major upgrade from v21 to v22 that could break `frontend-component-footer-edx`? That is, will supporting an older v21 version be compatible given it was previously using v22? If not, then I agree, expanding the support back to v21 is reasonable :)

Reply:
Ahh good call out. I was just replicating what was done in the [Open edX version](https://github.com/openedx/frontend-component-footer/blob/master/package.json#L68). This at least gives some reassurance. But looking at the [breaking changes for v22](https://github.com/openedx/paragon/releases/tag/v22.0.0), I don't see anything in there that would cause footer to break. The components called out in the release notes aren't used by frontend-component-footer-edx.